doc/installation_instructions/platform/cray.rst

```diff
@@ -0,0 +1,21 @@
+HPE Cray supercomputers
+=======================
+
+On certain HPE Cray machines, the SmartSim dependencies have been installed
+system-wide though specific paths and names might vary (please contact the team
+if these instructions do not work).
+
+.. code-block:: bash
+
+    module use -a /lus/scratch/smartsim/local/modulefiles
+    module load cudatoolkit/11.4 cudnn git-lfs
+
+    module unload PrgEnv-cray PrgEnv-intel PrgEnv-gcc
+    module load PrgEnv-gnu
+    module switch gcc/11.2.0
+
+    export CRAYPE_LINK_TYPE=dynamic
+
+Users should then proceed with their preferred way of installing SmartSim either
+:ref:`from PyPi <from-pypi>` or :ref:`from source <from-source>`.
```

Comment:
<img width="443" alt="Screen Shot 2023-02-28 at 10 09 49 AM" src="https://user-images.githubusercontent.com/30413257/221911173-49a4c80e-10bb-401f-9ad1-6ffebf269092.png">

Reply:
^this is what the links look like on my screen